### PR TITLE
feat: add @sbb-esta/lyne-components-react for react consumers

### PR DESCRIPTION
**Pull request by kyubisation**
This PR adds an additional package `@sbb-esta/lyne-components-react` intended for react consumers.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -66,6 +66,19 @@ jobs:
...
+      - name: 'Download artifacts'
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const fs = require('fs');
+            const version = require('./package.json').version;
+            const path = `${process.env.GITHUB_WORKSPACE}/react-library/package.json`;
+            const pkgJson = fs.readFileSync(path, 'utf8');
+            fs.writeFileSync(path, pkgJson.replace(/0.0.0-PLACEHOLDER/g, version), 'utf8');
+      - name: 'Release: Publish @sbb-esta/lyne-components'
```
Comment (jeripeierSBB) [inline .github/workflows/release.yml:78]:
```suggestion
      - name: 'Release: Publish @sbb-esta/lyne-components-react'
```

File: stencil.config.ts [hunk 2]
```diff
@@ -19,7 +20,7 @@ export const config: Config = {
       type: 'dist',
     },
     {
-      type: 'dist-custom-elements-bundle',
+      type: 'dist-custom-elements',
```
Comment (jeripeierSBB) [inline stencil.config.ts:23]: Can we so "easily" remove the "-bundle"? https://github.com/lyne-design-system/lyne-components/issues/946

File: stencil.config.ts [hunk 3]
```diff
@@ -2,6 +2,7 @@ import { inlineSvg } from 'stencil-inline-svg';
 import jestConfig from './.jest.config.js';
 import { sass } from '@stencil/sass';
 import { Config } from '@stencil/core';
+import { reactOutputTarget as react } from '@stencil/react-output-target';
```
Comment (jeripeierSBB) [inline stencil.config.ts:5]: seems not to be used?

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]

File: stencil.config.ts (hunk 2 above)
Comment (kyubisation) [reply stencil.config.ts:23]: I'm actually not sure.
@DavideMininni-Fincons If I remember correctly, quite a while ago you checked what the differences were. Do you remember any details?

File: stencil.config.ts (hunk 3 above)
Comment (kyubisation) [reply stencil.config.ts:5]: See line 38

File: stencil.config.ts (hunk 2 above)
Comment (DavideMininni-Fincons) [reply stencil.config.ts:23]: The issue about deprecating dist-custom-elements-bundle is still open. Link: https://github.com/ionic-team/stencil/issues/3136
It seems that there are still some bugs that Stencil team should fix.

Comment (kyubisation) [reply stencil.config.ts:23]: I'll revert it for now.

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**